tests/Client.Tests/Pages/Contracts/FavoriteButtonTests.cs

```diff
@@ -91,8 +91,8 @@ public void FavoriteButton_DoesNotChangeAppearance_WhenRequestFailed()
         string userName = "user";
         Contract contract = new();
 
-        MockHttp.When($"/api/v1/favorites/{userName}/{contract.Id}").Respond(req => new HttpResponseMessage(HttpStatusCode.OK)); // The contract is marked as favorite
-        MockHttp.When($"/api/v1/favorites").Respond(req => new HttpResponseMessage(HttpStatusCode.BadRequest)); // The contract does not get unmarked
+        MockHttp.When($"/api/v1/users/{userName}/favorites/{contract.Id}").Respond(req => new HttpResponseMessage(HttpStatusCode.OK)); // The contract is marked as favorite
```

Comment:
nit:
Did you try simplifying it like this?
```suggestion
        MockHttp.When($"/api/v1/users/{userName}/favorites/{contract.Id}").Respond(HttpStatusCode.OK); // The contract is marked as favorite
```